Resources/Prototypes/Entities/Objects/Power/lights.yml

```diff
@@ -95,16 +94,53 @@
 - type: entity
   parent: BaseLightbulb
   name: led light tube
+  description: A high power high energy bulb.
   id: LedLightTube
   components:
   - type: LightBulb
     bulb: Tube
     color: "#EEEEFF"
-    lightEnergy: 4
+    lightEnergy: 2.5
     lightRadius: 10
     lightSoftness: 0.9
     BurningTemperature: 350
-    PowerUse: 9
+    PowerUse: 25
+  - type: Sprite
+    sprite:  Objects/Power/light_tube.rsi
+    state: normal
+
+- type: entity
+  parent: BaseLightbulb
+  name: exterior light tube
+  description: A high power high energy bulb for the depths of space. May contain mercury.
+  id: ExteriorLightTube
+  components:
+  - type: LightBulb
+    bulb: Tube
+    color: "#9cbcff"
+    lightEnergy: 4.5
+    lightRadius: 12
+    lightSoftness: 0.5
+    BurningTemperature: 350
+    PowerUse: 100
   - type: Sprite
     sprite:  Objects/Power/light_tube.rsi
     state: normal
+    
+- type: entity
+  parent: BaseLightbulb
+  name: sodium light tube
+  description: A high power high energy bulb for the depths of space. Salty.
+  id: SodiumLightTube
+  components:
+  - type: LightBulb
+    bulb: Tube
+    color: "#FF7E00"
```

Comment:
I would like to propose `#FFAF38` which is[ taken from a blog](http://planetpixelemporium.com/tutorialpages/light.html) and adjusted to be slightly more orange.
![image](https://user-images.githubusercontent.com/7548248/150978890-5750ec10-da24-488c-a23a-b3b1cbcf1215.png)